nextflow.config

```diff
@@ -58,7 +58,7 @@ params {
 
     // Alignment
     aligner                    = 'star_salmon'
-    pseudo_aligner             = null
+    pseudo_aligner             = 'salmon'
```

Comment:
Should we have a pseudo aligner by default?

Reply:
also, unrelated, since we changed all `pseudo-alignment` to `pseudoalignement`, should we change the `pseudo_aligner` params to `pseudoaligner`?

Reply:
I think so, since pseudoalignment is not skipped by default

Reply:
... and on the params, I don't have a strong opinion so sure, if you want